libs/blocks/marquee/marquee.css

```diff
@@ -514,6 +551,12 @@
     max-height: 80px;
     max-width: 200px;
   }
+
+  .marquee .text .icon-area.icon-area-multiple img {
+    height: 40px;
+    max-height: 40px;
+    max-width: 114px;
+  }
```

Comment:
You set ` .marquee .text .icon-area.icon-area-multiple img` here for `min-width 1200lpx` and on lines 343 and 410 for the 600px media breakpoint and 900px media breakpoint. Shouldn't you be able to set it just for screens `min-width` 600px and it will apply to the other queries, or vice versa?

Reply:
Yes, it should work by setting it once. My thought process was that this would make it easier in the future to change the size on specific breakpoints, but I may be getting ahead of myself

Reply:
Ah I see. I think it is probably better to hold off adding this additional CSS until needed.